packages/ember-metal/lib/utils.js

```diff
@@ -799,7 +799,7 @@ export function inspect(obj) {
     return '[' + obj + ']';
   }
   // for non objects
-  if (typeof obj !== 'object') {
+  if (typeof obj !== 'object' && typeof obj !== 'symbol') {
```

Comment:
Lets cache the `typeof obj` result instead of calling it twice so we can avoid extra work.